include/quicly.h

```diff
@@ -241,6 +284,10 @@ struct st_quicly_context_t {
      * client-only
      */
     unsigned enforce_version_negotiation : 1;
+    /**
+     * if inter-node routing is used (by utilising quicly_cid_plaintext_t::server_id)
+     */
+    unsigned is_clustered : 1;
```

Comment:
Why not call this "has_node_id" or something more direct?

Reply:
Node_id exists even if the server is running standalone. It's just that it would always be set to zero. I might also argue that utilizing the node_id is a side-effect of running in cluster mode. Therefore `is_clustered` seems more natural to me.